src/com/goide/intentions/GoMoveToStructInitializationIntention.java

```diff
@@ -193,6 +224,19 @@ private static GoCompositeLit getStructLiteral(@NotNull GoReferenceExpression fi
     return ObjectUtils.tryCast(compositeLit, GoCompositeLit.class);
   }
 
+  @Nullable
+  private static GoCompositeLit getStructLiteral(@NotNull GoStatementImpl statement,
```

Comment:
It's `getOrCreateStructLiteral` and better create composite literal more clear.
you check was struct literal created again in `getData` : `isUnassigned(getSingleVarSpecByDefinition(previousStatement, structDefinition)))`